tests/wpt/tests/html/semantics/tabular-data/processing-model-1/span-limits.html

```diff
@@ -63,4 +63,46 @@
     assert_equals($("#d1").getBoundingClientRect().bottom,
                   $("#d2").getBoundingClientRect().bottom);
 }, "rowspan of 65535 must be treated as 65534");
+
+test(() => {
+    let td = document.createElement("td");
+    td.rowSpan = 5;
+    assert_equals(td.rowSpan, 5);
+
+    td.rowSpan = 0;
+    assert_equals(td.rowSpan, 0);
+
+    td.rowSpan = 1000;
+    assert_equals(td.rowSpan, 1000);
+
+    td.rowSpan = 65534;
+    assert_equals(td.rowSpan, 65534);
+
+    td.rowSpan = 65535;
+    assert_equals(td.rowSpan, 65534);
+
+    td.rowSpan = 555555;
+    assert_equals(td.rowSpan, 65534);
+}, "rowspan must be clamped to [0, 65534] when set via script");
+
+test(() => {
+    document.body.innerHTML += `
```

Comment:
This will replace all elements with a clone, which seems potentially problematic if e.g. `testharness.js` decides to create the results element before running this test.

Better use `insertAdjacentHTML` or just don't use JS to insert the code (move it to be with the other HTML).